docs/user_guide/capsules/index.md

```diff
@@ -45,7 +45,7 @@ The Capsule server is only supported on x86_64 Operating Systems
 
  * 2 Two Logical CPUs
  * 4 GB of memory
- * Disk space usage is similar to that of the main Katello server [Installation](./installation.html)
+ * Disk space usage is similar to that of the main Katello server [Installation](/docs/installation/index.html)
```

Comment:
Thanks @ntent-ashton - looks good but could you change this to be `/docs/{{ site.version }}/installation/index.html`?